### ISSUE TITLE
DataLogger.cpp typo

### ISSUE THREAD

**Issue by allthepies** [vitotai/BrewPiLess#23]
The CSS in the LogConfigHtml[] PROGMEM variable contains "1px solid whtie"

Presumably this should be "1px solid white"

**Comment by vitotai** [comment]
Yeah. I am thinking about changing it to "2px solid gray" or using border-style:groove.
